guides/05_tabular-data-science-and-plots/creating-a-dashboard-from-supabase-data.md

```diff
@@ -0,0 +1,122 @@
+# Creating a Dashboard from Supabase Data
+
+Tags: TABULAR, DASHBOARD, PLOTS 
+
+Supabase is a cloud-based open-source backend that provides a PostgreSQL database, authentication, and other useful features for building web and mobile applications. In this tutorial, we will show you how to read data from Supabase plot it in **real-time** on a Gradio Dashboard.
+
+**Prerequisites:** To start, you will need to have a free Supabase account, which you can sign up for here: https://app.supabase.com/
```

Comment:
make the link clickable 
`[https://app.supabase.com/](https://app.supabase.com/)`